main.go

```diff
@@ -24,8 +32,25 @@ func main() {
 	utils.Assert("error creating discord session", err)
 
 	discord.AddHandler(coreMessageHandler)
+
+	user, err := discord.User("@me")              //grabbing account information
+	utils.Assert("error retrieving account", err) //check if error occurred
+	botID = user.ID                               //botID is a variable set to the bots information
+	botname = "LANFestBot"
+
+	gameLibFile, err := ioutil.ReadFile("games.yml")
+	var yamlErr error
+	if err != nil {
+		yamlErr = yaml.Unmarshal(gameLibFile, &gameLib)
+	}
+	if err != nil || yamlErr != nil {
+		fmt.Printf("%s %s Error loading game library config file. LFG and game matching features will not work.\n", err, yamlErr)
+	}
+
+	discord.AddHandler(commandHandler) // a listener that when it picks up a message create it runs the function
+
 	discord.AddHandler(func(discord *discordgo.Session, ready *discordgo.Ready) {
-		err = discord.UpdateStatus(0, "Insaniquarium!")
+		err = discord.UpdateStatus(0, "with your heart <3")
```

Comment:
At this point, this code is beefy enough to be pulled into its own handler function and moved below (similar to how commandHandler is structured)

Reply:
My other PR takes care of splitting this out.